cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/widgets/RadioButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+export interface RadioButtonProps {
+    name: string,
+    value: string,
+
+    cdsLayout?: string,
+    checked?: boolean,
+    className?: string,
+    register: any,
+}
+
+// This is a convenience Component that allows a radio button to be CHECKED or not CHECKED using a property
+function RadioButton(props: Partial<RadioButtonProps>) {
+    if (props.checked) {
+        return <input className={props.className} cds-layout={props.cdsLayout} value={props.value} {...props.register(props.name)}
```

Comment:
Please check for indentation warnings with npm lint. We can adjust the rules to however the team would like but trying to establish some consistency. I think a double tab (8 space) indentation on a continued line will throw a warning.